src/app/states/editor/page.mdx

```diff
@@ -0,0 +1,64 @@
+export const metadata = {
+  title: 'editor',
+  description: 'An app state of the editor. This section explains how to fetch and manipulate the `editor` state.',
+}
+
+# editingNote
+
+An [app state](/guides/flux-architecture) of the editor.
+You can obtain the `editor` state and integrate it with React components.{{ className: 'lead' }}
+
+
+---
+
+## Data Structure
+
+<Row>
+  <Col>
+    <Properties>
+      <Property name="changed" type="boolean | undefined">
+        `true` if the note has been changed. 
+        The editor will save the note if `changed` is true when it is closed.
+        Otherwise it will be`undefined` if it has not been changed yet since the app launched.
+      </Property>
+      <Property name="needsUpdatingDate" type="boolean | undefined">
+        Whether the note's `updatedAt` field needs to be updated or not.
+      </Property>
+      <Property name="viewMode" type="EditorViewMode">
```

Comment:
@craftzdog This is the only one I am a little unsure of in the context of Typescript. Can you confirm if this is right or not?

Reply:
It can be fine as is, but more precisely, it should be `'preview' | 'sideBySide' | 'edit'`. Can you change it?